src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets.utils;
+
+import java.math.BigDecimal;
```

Comment:
Why are we utilizing BigDecimal?

Reply:
The reason is that pure double does not retain hundredth precision.
I used the double and it does not pass my test. 

Good resource to refer: https://stackoverflow.com/questions/322749/retain-precision-with-double-in-java

I got this mistake and found BigDecimal is a good solution to this.